packages/webpack-cli/src/webpack-cli.ts

```diff
@@ -675,10 +675,22 @@ class WebpackCLI implements IWebpackCLI {
       }
     }
 
+    let bracket: string | undefined;
+
     if (mainOption.type.size > 1 && mainOption.type.has(Boolean)) {
-      mainOption.flags = `${mainOption.flags} [value${mainOption.multiple ? "..." : ""}]`;
+      bracket = "[]";
     } else if (mainOption.type.size > 0 && !mainOption.type.has(Boolean)) {
-      mainOption.flags = `${mainOption.flags} <value${mainOption.multiple ? "..." : ""}>`;
+      bracket = "<>";
+    }
+
+    if (bracket?.length === 2) {
+      mainOption.flags =
+        mainOption.flags +
+        " " +
+        bracket[0] +
+        (mainOption.valueName || "value") +
+        (mainOption.multiple ? "..." : "") +
+        bracket[1];
```

Comment:
Can we rewrite it in one line using template literals, it is more readable?

Reply:
Sure, I can change that.

Yet, are you confident this will be more legible? I actually changed style because the operations performend within the template string very much bloated that line and, from my perspective, tended to render it less legible.

So, to be sure, would you please confirm that you may be happy with a line like this:

```ts
mainOption.flags = `${mainOption.flags} ${bracket[0]}${mainOption.valueName || "value"}${mainOption.multiple ? "..." : ""}${bracket[1]}`
```

Reply:
I am fine, just add a comment with an example, so everybody can undestand this :+1:

Reply:
Done.

While committing, the prettifier automatically added two line breaks to the template string, though.